stable/wordpress/templates/ingress.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | trimSuffix "-" -}}"
+  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | replace "." "_" | trimSuffix "-" -}}"
```

Comment:
`_` isn't valid right? Should probably be `-`.

Reply:
I think domain dots must be different from other parts.

Reply:
What I mean is that this replaces names like `test.test` to `test_test` which is also an invalid name in Kubernetes.